datasource/opennms-kafka/src/main/resources/inventory.groovy

```diff
@@ -0,0 +1,224 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2019 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2019 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+ package org.opennms.oce.datasource.opennms;
+
+import org.opennms.oce.datasource.common.ImmutableInventoryObject
+import org.opennms.oce.datasource.common.inventory.ManagedObjectType;
+import static org.opennms.oce.datasource.common.inventory.ManagedObjectType.*;
+
+import org.opennms.oce.datasource.common.inventory.TypeToInventory;
+import org.opennms.oce.datasource.opennms.EdgeToInventory
+import org.opennms.oce.datasource.opennms.EnrichedAlarm
+import org.opennms.oce.datasource.opennms.InventoryFromAlarm
+import org.opennms.oce.datasource.opennms.OpennmsMapper
+import org.opennms.oce.datasource.opennms.proto.InventoryModelProtos
+import org.opennms.oce.datasource.opennms.proto.InventoryModelProtos.InventoryObject;
+import org.opennms.oce.datasource.opennms.proto.InventoryModelProtos.InventoryObjects;
+import org.opennms.oce.datasource.opennms.proto.OpennmsModelProtos
+import org.opennms.oce.datasource.opennms.proto.OpennmsModelProtos.Node;
+import org.opennms.oce.datasource.opennms.proto.OpennmsModelProtos.TopologyEdge
+import org.opennms.oce.datasource.opennms.proto.OpennmsModelProtos.TopologyEdge.TargetCase
+
+import com.google.common.base.Strings;
+
+import groovy.util.logging.Slf4j
+
+@Slf4j
+class InventoryFactory {
+    
+    static InventoryObjects edgeToInventory(TopologyEdge edge) {
+        log.info("EdgeToInventory - edge: {}", edge);
```

Comment:
Can we convert these `log.info` statements to `log.debug` or even `log.trace`?

Reply:
done